tests/e2e_test/test_correctness.py

```diff
@@ -169,6 +177,12 @@ async def test_correctness(ray_env, shutdown_llumnix_service, check_log_exceptio
             if tensor_parallel_size == 2:
                 conftest.SKIP_REASON = "Simulator in TP = 2 will not be tested."
 
+        if not enable_migration:
+            if launch_mode != "global" or not enable_pd_disagg or migration_backend != "gloo" \
+                or tensor_parallel_size != 1 or enable_simulator:
+                conftest.SKIP_REASON = "Only test Migration=False in gloo migration backend for vLLM \
+                    when tensor parallelism=1 and prefill-decode disaggregation is enabled."
```

Comment:
the description is not accurate, should be 'Only test Migration=False for vLLM when ...